src/constants.rs

```diff
@@ -1,7 +1,5 @@
 use num_bigint::BigUint;
 
-pub const MODULUS_BIT_SIZE: usize = oblast::MODULUS_BIT_SIZE;
-
 pub fn get_modulus() -> BigUint {
```

Comment:
at this point, feels like it should be in `oblast`

Reply:
yeah, agreed